job/job.go

```diff
@@ -489,20 +497,44 @@ func (j *Job) AddToScheduler(cronRunner *cron.Cron) error {
 		j.Context.Infof("skipping scheduling")
 		return nil
 	}
-
 	j.Context.Logger.Named(j.GetResourcedName()).V(1).Infof("scheduled %s", schedule)
-	entryID, err := cronRunner.AddJob(schedule, j)
-	if err != nil {
-		return fmt.Errorf("[%s] failed to schedule job: %s", j.Label(), err)
+
+	if override, ok := j.GetProperty("jitter"); ok {
+		if parsed, err := duration.ParseDuration(override); err == nil {
+			j.Jitter = time.Duration(parsed)
+		}
 	}
-	j.entryID = &entryID
+
+	if j.Jitter == 0 {
+		entryID, err := cronRunner.AddJob(schedule, j)
+		if err != nil {
+			return fmt.Errorf("[%s] failed to schedule job: %s", j.Label(), err)
+		}
+		j.entryID = &entryID
+	} else {
+		delaydJob := cron.FuncJob(func() {
```

Comment:
I think the jitter should be once off at the start, i.e. delay the scheduling of the job by random(min(duration till next runtime, 1h))